master/buildbot/reporters/words.py

```diff
@@ -550,25 +550,22 @@ def buildFinished(self, build):
 
         if self.useRevisions:
             revisions = yield self.getRevisionsForBuild(build)
-            r = "Hey! build %s containing revision(s) [%s] is complete: %s" % \
+            r = "Build %s containing revision(s) [%s] is complete: %s" % \
                 (builderName, ','.join(revisions), results[0])
         else:
-            r = "Hey! build %s #%d is complete: %s" % \
+            r = "Build %s #%d is complete: %s" % \
                 (builderName, buildNumber, results[0])
 
         r += ' [%s]' % maybeColorize(build['state_string'],
                                      results[1], self.useColors)
-        self.send(r)
 
         # FIXME: where do we get the list of changes for a build ?
         # if self.bot.showBlameList and buildResult != SUCCESS and len(build.changes) != 0:
         #    r += '  blamelist: ' + ', '.join(list(set([c.who for c in build.changes])))
-
-        # FIXME: where do we get the base_url? Then do we use the build Link to
-        # make the URL?
-        buildurl = None  # self.bot.status.getBuildbotURL() + build
-        if buildurl:
-            self.send("Build details are at %s" % buildurl)
+        buildurl = self.master.status.getURLForBuild(builder['builderid'], buildNumber)
```

Comment:
well this is a weird choice of API change ... but ok i will do as you wish.
maybe add an Obsolete warning to old APIs?